app/Command/Compile.hs

```diff
@@ -72,25 +76,19 @@ compile PSCMakeOptions{..} = do
   printWarningsAndErrors (P.optionsVerboseErrors pscmOpts) pscmJSONErrors moduleFiles makeWarnings makeErrors
   exitSuccess
 
-warnFileTypeNotFound :: String -> IO ()
-warnFileTypeNotFound = hPutStrLn stderr . ("purs compile: No files found using pattern: " ++)
-
-globWarningOnMisses :: (String -> IO ()) -> [FilePath] -> IO [FilePath]
-globWarningOnMisses warn = concatMapM globWithWarning
-  where
-  globWithWarning pattern' = do
-    paths <- glob pattern'
-    when (null paths) $ warn pattern'
-    return paths
-  concatMapM f = fmap concat . mapM f
-
 inputFile :: Opts.Parser FilePath
 inputFile = Opts.strArgument $
      Opts.metavar "FILE"
   <> Opts.help "The input .purs file(s)."
 
-excludedFiles :: Opts.Parser FilePath
-excludedFiles = Opts.strOption $
+globInputFile :: Opts.Parser (Maybe FilePath)
+globInputFile = Opts.optional $ Opts.strOption $
+     Opts.long "source-globs"
```

Comment:
I think the `exclude-files` is a bit misnamed now that we have a feature that lets us specify a file to include globs (feels like it should be `exclude-globs`?) - anyhow I wonder if this new flag should be called `source-globs-file` instead, to note that it's a file containing globs, and not just globs.
```suggestion
     Opts.long "source-globs-file"
```

Reply:
Personally, I think that would be a good change to make, but it's breaking. We could do it in the `0.16.0` release.

Reply:
Then again... by updating the meta var from 'FILE' to 'GLOB' I think it does make more sense. So, maybe this change isn't needed?